btree.go

```diff
@@ -312,12 +312,16 @@ func (btree *btreeTable) parseCell() (cellInfo, error) {
 			if err != nil {
 				return cell, err
 			}
-			// FIXME(sbinet)
-			// - locate overflow-page (and following)
-			// - load payload
-			// - append into cell.Payload
 
-			panic("not implemented")
+			overflow, err := btree.readOverflow(cell.OverflowPage, overflowsz)
+			if err != nil {
+				return cell, err
+			}
+			cell.Payload = append(cell.Payload, overflow...)
+		}
+
+		if len(cell.Payload) != int(sz) {
+			panic(fmt.Sprintf("Tried to read %d payload bytes; got %d", sz, len(cell.Payload)))
```

Comment:
following Go practices, we don't capitalize error messages (at least, I try to follow that practice, now)
`s/Tried/tried/`

also, as for tests, we usually try to first tell what we got and then what we wanted.

so, I'd suggest writing it like so:
```go
panic(fmt.Errorf(read %d payload bytes instead of %d", len(cell.Payload), sz))
```

Reply:
done